ds248x/ds248x.go

```diff
@@ -179,31 +179,23 @@ func (d *Dev) ChannelSelect(ch int) error {
 	var err error = nil
 	switch d.isDS248x {
 	case isDS2482x100:
-		// if ch != 0 {
-		// 	err = errors.New("channel != 0")
-		// 	return fmt.Errorf("ds2482-100: error while selecting channel: %s", err)
-		// }
+		err = errors.New("unsupported (ds2482-100 has only channel 0)")
```

Comment:
Same here and for isDS2483; return errors.New("channel out of range").

Reply:
For ds2482-100 and ds2483 errors.New("channel out of range") would be misinformation. There is no channel to choose! My suggestion is either explicitly tell user that, either silently do nothing. I prefer the second by cause for ds2482-100 and ds2483 communication would continue on same channel any way. Migration from ds2482-800 to ds2482-100 would be simpler.

Reply:
"invalid channel" then. That's generic yet factual. On isDS2482x100 and isDS2483 it should only accept 0. The code can look like:

```
case isDS2482x100, isDS2483:
  if ch == 0 {
    return errors.New("invalid channel")
  }
  return nil
```